### PR TITLE
Improve debug output for some file system operations

### PR DESCRIPTION
**Pull request by daschuer**
This aims to help finding the root cause of issue https://github.com/mixxxdj/mixxx/issues/11778

### CODE REVIEW

File: src/library/parsercsv.cpp [hunk 1]
```diff
@@ -129,7 +133,7 @@ bool ParserCsv::writeCSVFile(const QString &file_str, BaseSqlTableModel* pPlayli
     if (!file.open(QIODevice::WriteOnly | QIODevice::Text)) {
         QMessageBox::warning(nullptr,
                 tr("Playlist Export Failed"),
-                tr("Could not create file") + " " + file_str);
+                tr("Could not create file") + " " + file_str + "\n" + file.errorString());
```
Comment (Holzhaus) [inline src/library/parsercsv.cpp:136]: This is possibly not localized and may look weird to inexperienced users. I think we should use the property-based API here and use the [detailed text](https://doc.qt.io/qt-5/qmessagebox.html#detailedText-prop) for this.

File: src/recording/recordingmanager.cpp [hunk 2]
```diff
@@ -189,11 +189,14 @@ void RecordingManager::setRecordingDir() {
         ConfigKey(RECORDING_PREF_KEY, "Directory")));
     // Note: the default ConfigKey for recordDir is set in DlgPrefRecord::DlgPrefRecord.
 
-    if (!recordDir.exists()) {
-        if (recordDir.mkpath(recordDir.absolutePath())) {
+    if (recordDir.exists()) {
```
Comment (Holzhaus) [inline src/recording/recordingmanager.cpp:192]: where did the `!` go?

File: src/recording/recordingmanager.cpp [hunk 3]
```diff
@@ -189,11 +189,14 @@ void RecordingManager::setRecordingDir() {
...
+        if (!recordDir.mkpath(recordDir.absolutePath())) {
```
Comment (Holzhaus) [inline src/recording/recordingmanager.cpp:193]: I think the exclamation mark is wrong here.

**Comment by daschuer** [comment]
Done.

**Review by Holzhaus — APPROVED** [review]
LGTM, thank you.

**Pull request merged**